src/styles/index.scss

```diff
@@ -1,3 +1,234 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700&display=swap';
+
+$dark: #060b35;
+$blue: #00acdc;
+$gray: #616070;
+$light-gray: #f3f3f3;
+$white: #fff;
+
+@mixin star {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
```

Comment:
Consider naming your mixins more descriptively. For instance, `star` could be `star-size` or `star-style` to give a clearer indication of its purpose.